spec/system/consumer/split_checkout_spec.rb

```diff
@@ -1114,11 +1114,8 @@
         let(:voucher) { create(:voucher, code: 'some_code', enterprise: distributor, amount: 6) }
 
         before do
-          # Add voucher to the order
```

Comment:
These comments might seem superfluous to some, but as a first-time reader of the code, I find them a helpful narration :)